src/media-groups.js

```diff
@@ -744,7 +744,7 @@ export const setupMediaGroups = (settings) => {
     mediaTypes.AUDIO.onTrackChanged();
   }
 
-  masterPlaylistLoader.on('mediachange', () => {
+  masterPlaylistLoader.on(['mediachange', 'mediachanging'], () => {
```

Comment:
Should we worry about this causing double aborts in some cases?

Reply:
🤔

Reply:
After some testing, this will cause a second abort on `mediachange` but the segmentLoader was already in an aborted state and nothing bad happens. After some more thought though I wonder if we should `stopLoaders` and `resync/reset` on `mediachanging` and then `startLoaders` on `mediachange` going to see how that works

Reply:
Not sure where you're at, but an alternative is to skip this change for the PR for right now

Reply:
yeah I gave myself until the end of today to try to solve it, just about to push up a commit that takes it out and fixes the tests.

Reply:
This took so long because this change seemed to cause other race condition issues and debugging it is quite difficult. Ultimately I think we still want some kind of fix, but it an edge case so we can probably just document it and skip for now.